src/status_im2/contexts/communities/home/style.cljs

```diff
@@ -42,3 +43,34 @@
    :left        0
    :padding-top top})
 
+;;;; CARD ANIMATION
+(def card-bottom-override {:margin-bottom 16}) ; Original 8 + 8 from tabs top padding
+(def card-height (+ 56 16)) ; Card height + its vertical margins
+(def card-total-height (+ card-height 8)) ; added 8 from tabs top padding
+(def card-opacity-factor (/ 100 card-height 100))
+
+(defn- value-in-range
+  "Returns `num` if is in the range [`lower-bound` `upper-bound`]
+  if `num` exceeds a given bound, then returns the bound exceeded."
+  [num [lower-bound upper-bound]]
+  (max lower-bound (min num upper-bound)))
+
+(defn set-animated-card-values
```

Comment:
I would prefer if this function was defined in the view namespace. This to me is a view hook. I think the guidelines (written or not I don't remember) agree that using `apply-animations-to-style` is acceptable in the `style` namespace, but setting shared values like this isn't.

I'm just concerned of the future where simple style namespaces are transformed into a mix of stateful code, because that's what animations do to the code 🤷🏼 Once this window is broken I bet things can get wild in the future. Where do we draw the line of what should be put inside `style` namespaces?

Reply:
Yes, I also was not sure.

Do you think I should create an `animation` namespace?

Reply:
The separate animation namespace is a rather interesting approach @ulisesmac. I think it makes a lot of sense when the `view` namespace is crowded with animation code. If it's just a handful of animations I think either approach (with/without separate ns) is good. Just extracting animation code to separate private functions goes a long way IMO, no need for namespaces.

I _usually_ lean on the side of less namespaces and more functions. I know it's not a popular opinion for some. Before we had standardized on using style + view namespaces, I wasn't a fan of that guideline, but it has grown on me, I find it acceptable now :) I'm usually not a fan of hard rules, like "always use a style ns", and so on.

For animation code on the other hand I think it can be quite useful, because animation code  REALLY gets in the way of understanding the structure of the components, the business rules involved and everything else. So much state...

Reply:
I moved them to the `view` ns, if we have more animations in the future for this namespace, I will consider creating an `animation` namespace.